packages/forklift-console-plugin/src/modules/Plans/views/details/utils/canDeleteAndPatchPlan.tsx

```diff
@@ -36,3 +36,23 @@ export const canDeleteAndPatchPlanMaps = (plan: V1beta1Plan) => {
 
   return canPatchNetworkMap && canDeleteNetworkMap && canPatchStorageMap && canDeleteStorageMap;
 };
+
+export const canDeleteAndPatchPlanHooks = (plan: V1beta1Plan) => {
```

Comment:
hooks should start with `hook...`
```suggestion
export const useDeleteAndPatchPlanHooks = (plan: V1beta1Plan) => {
```